CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,32 @@
+# HMIS API Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of behavior that may hinder the productivity of this environment include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others’ private information, such as a physical or electronic address, without explicit permission
+
+## Our Responsibilities
+
+HMIS API Specification contributors are responsible for clarifying the standards of positive behavior.
+
+The Department of Housing and Urban Development (HUD) has the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions.  
```

Comment:
```suggestion
The Department of Housing and Urban Development (HUD) and the HUD Data Lab has the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions.  
```

Reply:
Requesting that we include HUD Data Lab explicitly in this as well. The HUD Data Lab (ICF) is the party that will be responsible for most of the maintaining of this repository and everything in it.

Reply:
dun